pages/base.py

```diff
@@ -60,6 +61,13 @@ def login(self, email):
         self.selenium.get(login_link)
         self.wait.until(lambda s: self.is_user_loggedin)
 
+    def login_with_github(self, username, password, passcode):
+        self.click_sign_in_button()
+        auth0 = Auth0(self.selenium, self.base_url)
+        auth0.login_with_github(username, password)
+        tfa = TwoFactorAuthentication(self.selenium, self.base_url)
```

Comment:
If we need to do this every time, I'd put it all within `login_with_github`

Reply:
updated